docs/databases/connections/snowflake.md

```diff
@@ -6,9 +6,47 @@ redirect_from:
 
 # Snowflake
 
-## Schemas
+To add a database connection, click on the **gear** icon in the top right, and navigate to **Admin settings** > **Databases** > **Add a database**.
 
-Here you can specify which schemas you want to sync and scan. Options are:
+## Settings
+
+You can edit these settings at any time. Just remember to save your changes.
+
+### Display name
+
+What you want Metabase to call the database in its user interface.
+
+### Account name
+
+Enter your Account ID with the region that your Snowflake cluster is running on. E.g., `xxxxxxxxx.us-east-3.aws`.
+
+This field requires the alphanumeric account ID _with_ the region that your Snowflake cluster is running on. For example, if you're running Snowflake on AWS and your account URL is `https://az12345.ca-central-1.snowflakecomputing.com`, then the `Account` would be `az12345.ca-central-1.aws` (note the `.aws` suffix). There are some regions that don't need this suffix, so please [refer to the official Snowflake documentation](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#locator-formats-by-cloud-platform-and-region) for this
```

Comment:
```suggestion
Enter your Snowflake [Account Identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html). For example, if you're running Snowflake on AWS and your account URL is `https://az12345.ca-central-1.snowflakecomputing.com`:
- `<account_identifier>`: `az12345.ca-central-1`.
- `<cloud_platform>`: `aws`.

You'd enter `az12345.ca-central-1.aws` as the account name in Metabase.

> Not all regions require the cloud platform identifier. If you are in `us-west-2`, you would enter `az12345` as the account name. For the requirements per region, see [the official Snowflake's documentation](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#non-vps-account-locator-formats-by-cloud-platform-and-region).
```

Might as well update this, it was in the backlog already. For reference: https://discourse.metabase.com/t/time-out-snowflake-connection/23295.